src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -50,7 +50,7 @@ GcsServer::GcsServer(const ray::gcs::GcsServerConfig &config,
                            RayConfig::instance().gcs_server_rpc_client_thread_num()),
       raylet_client_pool_(
           std::make_shared<rpc::NodeManagerClientPool>(client_call_manager_)),
-      local_node_id_(NodeID::FromRandom()),
+      local_node_id_(NodeID::FromBinary(std::string(kUniqueIDSize, 0))),
```

Comment:
same question as https://github.com/ray-project/ray/pull/30460/files#r1072697672

Reply:
Here we make GCS address = 0x0. I should make it constant.
GCS should have fixed address since it can be restarted.